pokeapp/src/components/Layout.js

```diff
@@ -1,11 +1,11 @@
 import React from "react";
 
-const Layout = ({ Navigation, FilterBar, SideBar, DigitalCardBinder }) => {
+const Layout = ({ Navigation, FilterBar, SideBar, DigitalCardBinder, BadgeSideBar, isBadgeSideBar }) => {
   return (
     <div className="main-container">
       {Navigation}
       {FilterBar}
-      {SideBar}
+      {!isBadgeSideBar ? SideBar : BadgeSideBar}
```

Comment:
A toggle switch for this would be really useful so users dont have to go back to the navigation and click on pokedex or myPoke to look at the selected pokemon.